### PR TITLE
Update EffectPackages.ini with new repo

### PR DESCRIPTION
**Pull request by MaxG2D**
Added ReshadeSimpleHDRShaders repo to the list

### CODE REVIEW

**Comment by crosire** [comment]
It looks liks some of the code is based on work from others, but violating the license of that other work (by e.g. not reproducing the license text). Can't just slap a GPL on that I'm afraid.

**Comment by MaxG2D** [comment]
> It looks liks some of the code is based on work from others, but violating the license of that other work (by e.g. not reproducing the license text). Can't just slap a GPL on that I'm afraid.

Alright, will fix that this weekend hopefully, thank you

**Comment by MaxG2D** [comment]
> It looks liks some of the code is based on work from others, but violating the license of that other work (by e.g. not reproducing the license text). Can't just slap a GPL on that I'm afraid.

I've updated the license, hopefully it's all good now.

File: EffectPackages.ini [hunk 1]
```diff
@@ -340,3 +340,12 @@ TextureInstallPath=.\reshade-shaders\Textures
...
 EffectFiles=blur.fx,hyperblur.fx,anamorpho.fx
+
+[37]
+PackageName=Reshade Simple HDR Shaders by MaxG3D
+PackageDescription=Reshade HDR-Compatible shaders that focus on eye-candy effects and basic adjustments
+InstallPath=.\reshade-shaders\Shaders\MaxG3D
+TextureInstallPath=.\reshade-shaders\Textures
+DownloadUrl=https://github.com/MaxG2D/ReshadeSimpleHDRShaders/archive/refs/heads/main.zip
+RepositoryUrl=https://github.com/MaxG2D/ReshadeSimpleHDRShaders
+EffectFiles=HDRBloom.fx, HDRSaturation.fx, HDRMotionBlur.fx
```
Comment (crosire) [inline EffectPackages.ini]: Just need to get rid of those spaces after the commas, since the setup tool can't parse those, and we are good to go

Comment (MaxG2D) [reply EffectPackages.ini]: Done

**Comment by crosire** [comment]
Appears there is a texture name conflict in HDRMotionBlur.fx with MartysMods_SMAA.fx (both attempting to create a texture called `DepthTex`, but with different properties). Would be great if that could be rename, or wrapped in a namespace, to avoid this new conflict =)

**Comment by MaxG2D** [comment]
> Appears there is a texture name conflict in HDRMotionBlur.fx with MartysMods_SMAA.fx (both attempting to create a texture called `DepthTex`, but with different properties). Would be great if that could be rename, or wrapped in a namespace, to avoid this new conflict =)

It has been fixed in the latest release, sorry for all the trouble.

**Pull request merged**